servers/services/src/main/java/org/projectnessie/services/impl/NamespaceApiImpl.java

```diff
@@ -82,7 +82,7 @@ public Namespace createNamespace(String refName, Namespace namespace)
       throws NessieReferenceNotFoundException {
     Preconditions.checkArgument(!namespace.isEmpty(), "Namespace name must not be empty");
 
-    WithHash<NamedRef> refWithHash = namedRefWithHashOrThrow(refName, null);
+    ResolvedHash refWithHash = getHashResolver().resolveToHead(refName);
```

Comment:
The Namespaces endpoint is not supported in API v2, so relative hashes normally cannot make their way here. It's probably a moot point, but if there is an easier way to get HEAD, it might be worth using it in this class.

Reply:
The change here is purely cosmetic. `getHashResolver().resolveToHead(refName)` does exactly the same thing as `namedRefWithHashOrThrow(refName, null);`. But the new name is more meaningful imho. Do you see any other easier way to resolve the HEAD of a ref?